calculator.rb

```diff
@@ -0,0 +1,83 @@
+
+puts "*** Welcome to Natalie's Calculator! ***
+1. add(+)
+2. subtract(-)
+3. multiply(*)
+4. divide(/)
+-----------------
+Please chose an operator (name or symbol):"
+operator = gets.chomp
+
+# an array with all possible operators
+operators = ["+", "add", "-", "subtract", "*", "multiply", "times", "/", "divide"]
+
+until operators.include?(operator)
```

Comment:
This is a great use of until and include? to write clear and concise code to check for valid input.